### PR TITLE
create observability.md

### PR DESCRIPTION
**Pull request by swastiksingh22**
Signed-off-by: swastiksingh22 s.singh.swastik.98@gmail.com

### Describe your changes
I have added hindi version of observability.md. Thankyou I'll wait for your response.

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

File: content/hi/observability.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+title: ऑब्जर्वेबिलिटी(Observability)
+status: Completed
+category: संकल्पना
+---
```
Comment (anubha-v-ardhan) [inline content/hi/observability.md:5]:
```suggestion
---
title: ऑब्जर्वेबिलिटी (Observability)
status: Completed
category: संकल्पना
tags: ["property", "", ""]
---
```

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]
Hey @swastiksingh22 , The content of observability has changed on the English version. It would be nice if we can update this PR with latest content

**Comment by aj11anuj** [comment]
@swastiksingh22 gentle ping to implement necessary changes

**Review by aj11anuj — APPROVED** [review]
Lgtm 👍

**Review by kumarankit999 — APPROVED** [review]
looks good to me!

**Comment by aj11anuj** [comment]
@swastiksingh22 you need to change target branch from 'cncf: main' to 'cncf: dev-hi'.

**Comment by abhay-raj19** [comment]
Looks good to me 👍
Thanks @swastiksingh22 for contributing

**Review by abhay-raj19 — APPROVED** [review]

**Pull request merged**